R/cast.R

```diff
@@ -96,9 +96,10 @@
 #' vec_cast_common(factor("a"), factor(c("a", "b")))
 #' vec_cast_common(factor("a"), Sys.Date(), .to = list())
 vec_cast <- function(x, to) {
-  if (is.null(x) || is.null(to)) {
-    return(x)
-  }
+  return(.Call(vctrs_cast, x, to))
+  UseMethod("vec_cast", to)
```

Comment:
I assume you leave the `UseMethod()` to trick roxygen?

Reply:
Yes, or for other static analysis purposes. That's going to be a common trick in the package so I didn't add a comment as I did for `vec_type2()`.